helix-view/Cargo.toml

```diff
@@ -39,6 +39,8 @@ log = "~0.4"
 
 which = "4.2"
 
+ec4rs = "1.0.0-rc.1"
```

Comment:
Not very familiar with ec4rs so took a quick look. I find the API and naming a bit weird, at least not like the other rust crates, like `get_idxes` https://github.com/TheDaemoness/ec4rs/blob/1c5134139e8179fec1323bb6a176868df68fc5f8/src/properties.rs#L25, I think it might be better for us to have editorconfig code within helix since it should not be that huge.

Reply:
Can you first make it 0.1 rather than major release straight away, the API could be improved I think, not sure if we want to stagnant it in the first try.

Reply:
Oh the API definitely has room for improvement before the `1.0.0` release. I've listed the changes I definitely want to make going forward [here](https://github.com/TheDaemoness/ec4rs/issues/1); if there's anything that stands out to you that's either missing or that you disagree with as a change, please let me know there!